packages/ndla-ui/src/Footer/Footer.tsx

```diff
@@ -117,21 +116,20 @@ const StyledLanguageWrapper = styled.div`
 type Props = {
   children: React.ReactNode;
   lang: 'nb' | 'nn' | 'en';
-  t(arg: string, obj?: { [key: string]: string | boolean | number }): string;
-  links?: {
-    email: string;
```

Comment:
Var ikkje denne i bruk? Og heller ikkje email?

Reply:
`links` blir sendt fra `Footer.tsx` som props til `FooterLinks.tsx` og brukt slik:
```
            {links.map(link => (
              <StyledSocialMediaLinkWrapper key={link.to}>
                <StyledSocialMediaIcon>{link.icon}</StyledSocialMediaIcon>
                <StyledSafeLink to={link.to}>
                  {link.text}
                  <Forward />
                </StyledSafeLink>
              </StyledSocialMediaLinkWrapper>
            ))}
```
Ser ut som bare `to, icon, text` blir brukt. Hele poenget med endringen var at typescript klagde på at `links` er en array i `FooterLinks.tsx` men definert som et objekt i `Footer.tsx` så jeg endret de til array for at det skal stemme med loopen i koden over.